grakn-engine/src/main/java/ai/grakn/engine/backgroundtasks/distributed/ClusterManager.java

```diff
@@ -200,8 +202,12 @@ private void startZookeeperConnection() throws Exception{
      * Close all connections to Zookeeper
      */
     private void stopZookeeperConnection(){
-        cache.close();
-        failover.close();
+        if (cache!=null) {
```

Comment:
@fppt  Only one peer in the cluster will have a scheduler running. There is framework that we use for electing a "leader" (a scheduler for the tasks).